data-model/SDP-sample-records/textual-artifact_sample1.json

```diff
@@ -0,0 +1,95 @@
+{
+  "id": "ark:/21198/adf20c1f",
+  "type": "Textual Artifact",
+  "label": "Gospels",
+  "extent": ["f. 1-152"],
+  "work": "https://sinaimanuscripts/library.ucla.edu/work/1",
+  "associated_persons": [
+    {
+      "type": "attributed-author",
+      "id": "https://sinaimanuscripts/library.ucla.edu/person/42",
+      "attested_name_NS": "ܕܩܕܝܫܐ ܡܪܝ ܝܘܚܢܢ",
+      "attested_name": "Qadisho Mar Yuhanon",
+      "source": "Final rubric on f. 152v"}
+  ],
+  "paratexts": [
+    {
+      "type": "explicit",
+      "locus": "f. 152v",
+      "transcription": {"syr": ["ܐܝܬ ܕܝܢ ܐܦ ܐܚܪ̈ܢܝܬܐ ܣܓ̈ܝܬܐ ܕܥܒܕ ܝܫܘܥ ܐܝܠܝܢ ܕܐܠܘ ܚܕܐ ܚܕܐ ܡܬܟܬܒܢ ܗܘ̈ܝ. ܐܦ ܠܐ  ܗܘ ܥܠܡܐ ܐܝܟ ܕܣܒܪ ܐܢܐ. ܣܦܩ ܗܘܐ ܠܟܬܒ̈ܐ ܕܡܬܟܬ̈ܒܝܢ "]},
+      "translation": {"en": ["There were also many other things which Jesus did that, were each one written, I believe, not even the world would be sufficient for the books which would be written."]}
+    },
+    {
+      "type": "final rubric",
+      "locus": "f. 152v",
+      "transcription": {"syr": ["ܫܠܡ ܐܘܢܓܠܝܘܢ ܕܩܕܝܫܐ ܡܪܝ ܝܘܚܢܢ ܕܡܠܠ ܝܘܢܐܝܬ ܒܐܦܣܘܣ"]},
+      "translation": {"en": ["Finished is the Gospel of Saint John which he spoke in Greek in Ephesus."]}
+    }
+  ],
+  "defective": true,
+  "condition":
+  {
+    "note": ["Syriac 2A originally belonged together with ms. Chabot 12 (Milan, Biblioteca Ambrosiana)."],
+    "features": []
+  },
+  "contents":
+  {
+    "note": ["Incomplete at the beginning, missing portions of the Gospel of Matthew"],
+    "features": [],
+    "toc": [
```

Comment:
For each of these items, we could also include associated paratextual content (e.g. if there's marginal commentary only on the Gospel of Mark) or attested author information, etc. if it's interesting? I think we do want multiple, more or less structured, ways to define the sub-items within a textual artifact. I agree that we should define an artifact based on what we can tell of the intention of the producers of the ms (i.e., which texts or groups of texts are represented in the ms as a cohesive unit?). And that artifacts shouldn’t ‘nest’, even if they contain semi-distinct intellectual contents as in the case of the Gospels, or a liturgical service made up of different prayers and hymns. The artifact is the Gospels, even if we want to include information about where the individual books start and end.

Still, we could use the ‘toc’ field or something akin to it to provide structured information about the sub-items contained within an artifact. For instance, we could gather information about which hymns, sermons, etc. make up a liturgical service. These could be linked to work records, as a user might be interested either in the service as a whole or the witnesses for a particular hymn that features in it.

The items in the ‘toc’ field could have most of the same fields as a textual artifact, such as associated persons, extent, paratexts, etc. (This would be useful for providing attested information, name variants, etc. for person and work records). But we wouldn’t make them full-on textual artifact records. One potential complication might be parts of an artifact that themselves have sub-parts (a liturgical book with services for the whole year comes to mind -- each service would be an item in the 'toc' but would consist of sub-items for the parts of the service, etc.).

I think another benefit of having ‘contents’ with various fields within it allows us to capture different levels of detail, depending on how much time, interest, or expertise is available to a given textual artifact.

Reply:
I will move this to its own issue for further discussion